xls/build_rules/xls_macros.bzl

```diff
@@ -275,6 +275,7 @@ def xls_dslx_verilog_build_and_test(
     build_test(
         name = "__" + name,
         targets = [":" + name],
+        **kwargs,
```

Comment:
It seems maybe worth grabbing the `tags` argument specifically; I'm not sure we want to pass all kwargs through to the build_test.

Reply:
Yeah, you are right. I've changed this, thanks for the suggestion!